packages/angular-test-app/src/preview-examples/toast-custom.ts

```diff
@@ -13,14 +13,7 @@ import { ToastService } from '@siemens/ix-angular';
 @Component({
   selector: 'app-example',
   template: `
-    <div>
-      Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
-      eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
-      voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
-      clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit
-      amet.
-    </div>
```

Comment:
Why did you remove the this block?

Reply:
To be consistant with the other examples.